flaskr/gs_mock.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+from flask import request, jsonify
+import numpy as np
+from pydantic import BaseModel, Field
```

Comment:
using pydantic to add type validation (as python only _hints_ types, not validation)